doc/Interface.md

```diff
@@ -0,0 +1,44 @@
+The main interface is a sprite-based dimetric world with an overlaid window system. The dimetric world is a standard isometric-like game world. Clicks are registered on a sprite-by-sprite basis; if a user clicks on a guest on a path on a terrain tile then it is the guest which is identified as the target of the clicking. The windowing system is a standard layered window system.
```

Comment:
Ugh, doc/usermanual.md ?  Eventually, probably doc/usermanual/*  but that may be too early